src/index.html

```diff
@@ -15,8 +15,73 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          class="imagem"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class name 'imagem' could be more descriptive of the content or its role, such as 'logo-image'.

Reply:
+- justo